docs/source/schema/index.html.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Schema Reference
+
+language_tabs: # must be one of https://git.io/vQNgJ
+  - xml
+  - json
+
+includes:
+  - schema/catalogXML
+  - schema/catalogJSON
+  - schema/profileXML
+  - schema/profileJSON
+
```

Comment:
@wendellpiez the `includes` paths are wrong since the files are now in subdirectories ... they should be as follows:

```
  - schema/xml/catalog
  - schema/json/catalog
  - schema/xml/profile
  - schema/json/profile
```

@JJediny ^ ... this explains why my middleman docs generation was hanging yesterday

Reply:
:+1: